topostats/io.py

```diff
@@ -497,9 +497,9 @@ def load_spm(self) -> tuple:
             # trying to return the error with options of possible channel values
             labels = []
             for channel in [layer[b"@2:Image Data"][0] for layer in scan.layers]:
-                channel_name = channel.decode("latin1").split(" ")[1][1:-1]
-                # channel_description = channel.decode('latin1').split('"')[1] # incase the blank field raises quesions?
-                labels.append(channel_name)
+                # channel_name = channel.decode("latin1").split(" ")[1][1:-1]
```

Comment:
```suggestion

```

Reply:
Ok, two approaches @derollins 

1) You can if you want directly commit the above suggestion to this Pull Request. The Pull Request has been approved already, this may remove the approval if so it will require approving again. This change won't be reflected on your local branch. That isn't necessarily a problem though as you probably won't touch this branch again after its merged and can delete it.

2) Make the change on your local branch (on your laptop) and push it to GitHub, it will be tagged onto and included in this Pull Request. I would expect this to remove the approval too and it would then require re-approval.

Once re-approvals have been made you can go ahead and "Merge" the pull request, on doing so you'll be offered the opportunity to delete the branch on GitHub which is perfectly fine to do.